src/components/Assemblies/Members/Header.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+	import { Breadcrumb, BreadcrumbItem, Search } from 'carbon-components-svelte';
+	import Download from 'carbon-icons-svelte/lib/Download.svelte';
+	import TableSplit from 'carbon-icons-svelte/lib/TableSplit.svelte';
+	import type {
+		AssemblySummary,
+		GroupByTab
+	} from '../../../routes/assemblies/[id]/members/[groupby]/+layout';
+
+	export let data: {
+		assembly: AssemblySummary;
+		groupByTabs: GroupByTab[];
+	};
+
+	$: ({ assembly } = data);
+
+	export let searchQuery = '';
+	export let mounted = false;
+	$: startedYear = assembly?.startedAt ? assembly.startedAt.getFullYear() + 543 : null;
```

Comment:
Use `toLocaleDateString` or `dayjs` instead of manually + 543 to avoid the magic number.